src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/corpus_pruning_task_test.py

```diff
@@ -395,7 +392,6 @@ def setUp(self):
         job='libfuzzer_asan_job2',
         last_run=datetime.datetime.now()).put()
 
-    environment.set_value('USE_MINIJAIL', True)
```

Comment:
I assume minijail was used because untrusted used to be used with it all the time. Since this is not the case anymore, I've removed it.